packages/auth-web-sdk/src/api.ts

```diff
@@ -6,8 +6,79 @@ import {
   RequestTokenResponseData,
   GetLogInUrlArgs,
   GetLogOutUrlArgs,
+  GrantType,
+  RequestClientTokenArgs,
+  RequestClientTokenResponseData,
 } from './types'
 
+export const requestClientToken = async ({
+  clientId,
+  clientSecret,
+  issuerUrl,
+  scopes,
+}: RequestClientTokenArgs) => {
+  try {
+    const body = {
+      grant_type: GrantType.ClientCredentials,
+      scope: scopes.join(' '), // a string with space seperated scopes
```

Comment:
I think the code is self-explanatory enough to get rid of the comment 🙂

Reply:
updated the comment to explain more about why I left it - `scope` is a bit misleading as it sounds like it only takes one scope as a param vs multiple.